packages/components/modules/notifications/NotificationsPopover/__storybook__/stories.tsx

```diff
@@ -0,0 +1,104 @@
+import type { Meta, StoryObj } from '@storybook/react'
```

Comment:
![image](https://github.com/user-attachments/assets/f47895ef-e5bb-43e4-a294-1ac65a5c5b7d)
i think we need to mock the data for the drawer as well, now is just loading

Reply:
we decided to mock only the first call, and I think that's why we are not rendering the list. Now that we have `queueOperationResolver` I'll try using it